internal/pkg/services/iaas/utils/utils.go

```diff
@@ -16,6 +17,18 @@ type IaaSClient interface {
 	GetNetworkAreaRangeExecute(ctx context.Context, organizationId, areaId, networkRangeId string) (*iaas.NetworkRange, error)
 }
 
+func GetPublicIP(ctx context.Context, apiClient IaaSClient, projectId, publicIpId string) (ip, associatedResource string, err error) {
```

Comment:
The second return argument is not used in my PR, but I took this util function from [this open PR](https://github.com/stackitcloud/stackit-cli/pull/513/files#diff-4667d7303e488ebb891426ede3ae2da4ae33eab7629ca8e7145f37f4596598b8) from 
@GokceGK where it is needed